neurons/validators/validator.py

```diff
@@ -266,13 +271,20 @@ def __init__(self):
 
             self.masking_functions = [
                 self.blacklist,
-                task_validator,
                 relevance_model,
                 self.diversity_model,
                 nsfw_model,
             ]
+
+            self.penalty_functions = [
+                TaskValidationPenaltyModel(max_penalty=0.1),
+                SentenceMatchPenaltyModel(max_penalty=0.1),
+                KeywordMatchPenaltyModel(max_penalty=1),
```

Comment:
are we keeping this pseudo-filter model blacklist in for the next release?

Reply:
A pseudo-filter is preferable to having no filter at all, especially when it produces tangible effects on the network. We can consider removing it once n-gram matching demonstrates its efficacy.